content/en/docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -278,6 +278,36 @@ kubectl apply -f https://docs.projectcalico.org/v3.1/getting-started/kubernetes/
 kubectl apply -f https://docs.projectcalico.org/v3.1/getting-started/kubernetes/installation/hosted/canal/canal.yaml
 ```
 
+{{% /tab %}}
+
+{{% tab name="Cilium" %}}
+Cilium is open source software for providing and transparently securing network connectivity and loadbalancing between application workloads.
+
+Passing `--pod-network-cidr` option to `kubeadm init` is not required, but highly recommended.
+
+These commands will deploy Cilium with its own etcd managed by etcd operator.
```

Comment:
-> These commands deploy Cilium on its own instance of etcd, managed by etcd Operator.

(Q: is this a different deployment model from other networking solutions? If so, can you say more about how it works and why it's important?)

Reply:
@Bradamant3 Cilium uses etcd as a key-value store for assigning and synchronizing security identities between nodes (https://cilium.readthedocs.io/en/v1.2/kvstore/#layout).

We used to suggest that users should connect to existing Kubernetes-backing etcd instance, but this is highly discouraged in Kubernetes community, so we ended up with this set of instructions which deploys Cilium-specific etcd with TLS enabled.

Some CNIs are also deploying their own etcd (like Romana: https://raw.githubusercontent.com/romana/romana/master/containerize/specs/romana-kubeadm.yml), but TLS is not enabled by default. Some other CNIs are connecting to Kubernetes Custom Resources exclusively, but Cilium is unable to do so yet (this is tracked here: https://github.com/cilium/cilium/issues/4057).

I can delete the "etcd" part in this documentation, but I thought it might be relevant for some users.

Reply:
i would modify this to:
```
These commands will deploy Cilium with its own etcd, managed by an etcd operator.
```
but ideally there should be a bit more detail here.